certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -1614,13 +1680,17 @@
       }
     },
     "AdditionalRecipientData": {
+      "description": "an object has additional parameters of a recipient.",
```

Comment:
nit : reword this. "An object that represent additional parameters of a recipient."